src/coreclr/tools/Common/JitInterface/CorInfoImpl.cs

```diff
@@ -3656,7 +3658,26 @@ private BlockType findKnownBlock(void* location, out int offset)
         private static RelocType GetRelocType(TargetArchitecture targetArchitecture, ushort fRelocType)
         {
             if (targetArchitecture != TargetArchitecture.ARM64)
```

Comment:
Could you please refactor this into a switch?
```
switch (targetArchitecture)
{
case TargetArchitecture.ARM64:
    ...
case argetArchitecture.LoongArch64:
    ....
default:
    ...
}
```

Reply:
OK, Thanks.